source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRTextBlockRenderer.mm

```diff
@@ -116,6 +116,11 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
         lab.textContainer.lineBreakMode = NSLineBreakByTruncatingTail;
         lab.attributedText = content;
         lab.accessibilityLabel = content.string;
+        // if accessibility label is the same as accessibility value, clear accessibility value
+        // this prevents the same content from being repeated twice in voiceover
+        if (lab.accessibilityValue != nil && [lab.accessibilityValue isEqualToString:lab.accessibilityLabel]) {
+            lab.accessibilityValue = @"";
```

Comment:
yeah with a value and a label it can be repeated. This should work fine but maybe setting it to nil would imply that we don't want to use it more?

Reply:
I initially tried that, but setting it to nil doesn't seem to clear the value (voice over still announces twice)